app/views/admin/tasks/_form.html.erb

```diff
@@ -1,12 +1,29 @@
+<% translation_base = "#{claim.policy.to_s.underscore}.admin.task_questions.#{task_name}" %>
 <%= form_with model: [:admin, @claim, @claim.tasks.where(name: task_name).first_or_initialize] do |f| %>
   <div class="govuk-form-group">
     <fieldset class="govuk-fieldset govuk-!-margin-bottom-6">
       <legend class="govuk-fieldset__legend govuk-fieldset__legend--l ">
         <h3 class="govuk-heading-l">
-          <%= I18n.t("#{claim.policy.to_s.underscore}.admin.task_questions.#{task_name}") %>
+          <%= I18n.t("#{translation_base}.title", claim_full_name: @claim.full_name) %>
         </h3>
       </legend>
 
+      <%
+          I18n.t(
+            "#{claim.policy.to_s.underscore}.admin.task_questions.#{task_name}.body",
+            default: nil,
+            link: link_to(
+              I18n.t("#{translation_base}.notes"),
+              admin_claim_notes_path(@claim)
+            )
+          )&.html_safe.tap do |translation| %>
+        <% break unless translation %>
```

Comment:
@nomasprime Is this to cater for the fact there are no translations for `early_career_payments.task_questions`?

Is there are reason there aren't any entries for this key?

Reply:
There's no presenter for ECP.

Reply:
@nomasprime out of curiosity purely, why is there no presenter for ECP?